src/NUnitFramework/framework/Constraints/StringConstraint.cs

```diff
@@ -98,12 +98,12 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
 
             return new ConstraintResult(this, actual, Matches(actualAsString));
         }
-
+               
```

Comment:
It would be cool if this file didn't have to show up as changed in this PR. By the way, feel free to squash and rebase however suits you.